src/components/Navbar/Navbar.js

```diff
@@ -1,17 +1,40 @@
 import React from "react";
-import './Navbar.scss';
 import {Link} from 'react-router-dom';
+import SearchBar from '../SearchBar/SearchBar.js';
+import Button from '../Button/Button.js';
+import Logo from '../Logo/Logo';
+import './Navbar.scss';
```

Comment:
organize your imports.
.scss navbar file meed to be after React import
js file play synchrony